clients/devtools-entry.js

```diff
@@ -36,6 +36,7 @@ function getDefaultConfigForCategories(categoryIDs) {
 function runLighthouseInWorker(port, url, flags, categoryIDs) {
   // Default to 'info' logging level.
   flags.logLevel = flags.logLevel || 'info';
+  flags.channel = 'cdt';
```

Comment:
do we have a good place to test any/all of these? It's going to be difficult to notice if something is accidentally falling back to the node default, but I don't know if we have a good place to check

Reply:
Added a test case for the Chrome extension.